x-pack/plugins/security/server/plugin.ts

```diff
@@ -98,6 +98,10 @@ export interface PluginStartDependencies {
  * Represents Security Plugin instance that will be managed by the Kibana plugin system.
  */
 export class Plugin {
+
+
+  console.log('Hello everybody');
```

Comment:
```suggestion
  console.log('Hello, everybody');
```

Reply:
If we're going to give suggestions, I would like this to be moved to a function, outside the class definition.

Reply:
Can we also run the string through [18n](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md) please

Reply:
I'm torn on this to be honest. From one hand I understand why you made this change here since it's a clear candidate to be a platinum security feature, but on the other hand if we move it to the Core (e.g. use it instead of `Kibana server is not ready yet`) and greet everyone irrespective to the distribution and license they use we may benefit from it much more in the long term :man_shrugging: